x-pack/solutions/security/plugins/security_solution/common/siem_migrations/model/rule_migration.schema.yaml

```diff
@@ -444,3 +448,23 @@ components:
             updated_by:
               type: string
               description: The user who last updated the resource
+
+    RuleMigrationLastExecution:
+      type: object
+      description: The last execution of the rule migration task.
+      properties:
```

Comment:
Currently all properties are optional. Is this expected? It might be that we expect the entire object to be optional, not individual properties.